README.md

```diff
@@ -6,171 +6,190 @@ collaboration on unsolved problems in theoretical computer science, so
-  can open).  Links to other messages via specical coauthor:xxx syntax.
-  Drag messages (via their arrow icon) into other messages to make such links.
-
-* **Files** (another type of message) can be attached to other messages, as
-  another type of reply.  You can click on the Attach button to select a file
-  to attach, or drag the file from the operating system onto the Attach
-  button.  Similarly, files can be modified by clicking the Replace File
-  button or dragging a file onto that button.
-  File messages can have title and body too; title defaults to the filename.
-  Image/video files (including PNG, JPEG, SVG, MP4) are displayed inline.
-  (In the future, they and other visual file types such as PDF will be
-  rendered by some kind of thumbnails.)
-
-* Messages can start **Unpublished**, or after publication, **Deleted**;
-  in either state, the message is hidden from people who are not authors
-  (an *author* is someone who has edited the message), @-mentioned
-  (via `@username`), or superusers.
-  The default published state is initially true (so everyone sees the new
-  message and live updates immediately), but can vary by user (e.g., if they
-  are "shy" and only want to post finished thoughts).
-
-* Threads can be marked as allowing **public** replies only (the default, for
-  maximum collaboration), **private** replies only (useful for solved
-  problems/puzzles, to prevent accidentally spoiling of the fun), or
-  **public and private** replies (useful for feedback on lectures, for example,
-  which can have varying relevance to the entire group).  Replies to replies
-  inherit the public/private state of their parent.  Superusers can
-  see all the messages and change them between public and private.
-  Private messages can @-mention another user (via `@username` in the body)
-  to allow them to see and jointly edit the message.
-
-* **Email notifications** for subscribed threads, clustering together all
-  updates since the last email, with a maximum lag a specified number of
-  hours or minutes (default 1 hour).
-  Each user can specify in Settings whether they are, by default, subscribed
-  to all threads or none, both globally and local to each group.
-  Either way, the default can be overridden in the group view using the
-  checkbox on the right (checked means "subscribed").
-  Users can choose in Settings whether to receive notifications
-  about their own edits.
-
-* **Time travel**: You can drag through history and see past versions.
-  In general, there should be good, automatic history tracking of everything,
-  including a not-yet-visible reparenting feature.
-
-* **Superuser operations** (only for superusers):
-  * Import from LaTeX document with figures attached as a ZIP file
-  * Import from osqa's XML dump, including old edit history
-  * Superdelete (permanently destroying a message including its history)
-  * Setting the default sort for a group
+## Features so far ##
+
+### Live updates ###
```

Comment:
Allows to point to documentation sections via `#{title}` e.g. `#live-updates` or `#groups`.